pkg/apis/extensions/v1beta1/register.go

```diff
@@ -60,6 +58,7 @@ func addKnownTypes() {
 		&ListOptions{},
 		&ConfigMap{},
 		&ConfigMapList{},
+		&v1.DeleteOptions{},
```

Comment:
Because clients need a consistent view of the world.  Different groups
evolve at different times - when we lock and create a version we need
to identify the recognized options.  There can be no consistency
outside of a version.  All these lines say is is that this go type
corresponds to a particular schema.

We absolutely, totally, 100% want a DeleteOptipns and ListOptions in
every group.